### PR TITLE
app-crypt/johntheripper: fix building with clang

### PR DESCRIPTION
**Pull request by austin987**
Gentoo-Bug: https://bugs.gentoo.org/554392

### CODE REVIEW

File: app-crypt/johntheripper/johntheripper-1.7.9-r11.ebuild [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+				-i src/Makefile || die
+		fi
+
+		# fix typo in jumbo patch
+		sed -i 's:All15:All5:' run/john.conf || die
+		# fix compile on ppc (only needed for jumbo-7)
+		eapply "${FILESDIR}/${P}-ppc-compile-fix.patch"
+	fi
+
+	cd src
```
Comment (monsieurp) [inline app-crypt/johntheripper/johntheripper-1.7.9-r11.ebuild]: missing die

File: app-crypt/johntheripper/johntheripper-1.7.9-r11.ebuild [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+	if use mozilla; then
+		sed -i -e '/HAVE_NSS =/s/^#*//' -e 's/NSS_override//' Makefile || die
+	fi
+}
+
+src_compile() {
+	local OMP
+
+	use custom-cflags || strip-flags
+	echo "#define JOHN_SYSTEMWIDE 1" >> config.gentoo
```
Comment (monsieurp) [inline app-crypt/johntheripper/johntheripper-1.7.9-r11.ebuild]: I'd wrap these three echo calls in a here doc and append a die to the end.

**Comment by austin987** [comment]
@monsieurp updated, thanks for feedback

**Comment by austin987** [comment]
Fixed by https://github.com/gentoo/gentoo/commit/f44215bc85605856c871874d5dd3837be93838cf

**Pull request closed without merging**